IoTHubManager/Constants.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
```

Comment:
Currently, I am having this file separate from others. Will merge with the file under Helpers proj, once it is available.